<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room</class>
 <widget class="QWidget" name="Room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>790</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>790</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>790</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(225, 229, 228);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>1000</width>
     <height>730</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>730</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>730</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="videoButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>700</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(163, 183, 160);
font: 11pt &quot;黑体&quot;;
border-radius:7px;</string>
    </property>
    <property name="text">
     <string>选择视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kickButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>700</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(163, 183, 160);
font: 11pt &quot;黑体&quot;;
border-radius:7px;</string>
    </property>
    <property name="text">
     <string>踢出成员</string>
    </property>
   </widget>
   <widget class="OpenglWidget" name="openglWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>680</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>310</y>
     <width>290</width>
     <height>410</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>290</width>
     <height>410</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>290</width>
     <height>410</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:7px;
background-color:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>730</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:7px;
background-color:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>730</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(163, 183, 160);
font: 11pt &quot;黑体&quot;;
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>70</y>
     <width>290</width>
     <height>230</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>290</width>
     <height>230</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>290</width>
     <height>230</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:7px;
background-color:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inviteButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>30</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(163, 183, 160);
font: 11pt &quot;黑体&quot;;
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>邀请好友</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leaveButton">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>30</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(163, 183, 160);
font: 11pt &quot;黑体&quot;;
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>退出房间</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenglWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">openglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
